spec/models/manageiq/providers/hawkular/middleware_manager/hawkular_helper.rb

```diff
@@ -12,11 +13,13 @@ def test_end_time
 end
 
 def test_hostname
-  'hservices.torii.gva.redhat.com'.freeze
+  # 'hservices.torii.gva.redhat.com'.freeze
+  'localhost'.freeze
 end
 
 def test_port
-  80
+  # 80
```

Comment:
It is just a reminder that hservices.torii.gva.redhat.com is on port 80. I can remove the comment if you want.